packages/data-point/examples/reducer-conditional-operator.js

```diff
@@ -25,6 +26,8 @@ dataPoint.addEntities({
   }
 })
 
+mockRequests()
```

Comment:
Could use that same comment highlighting it's mocking the server request (for people unfamiliar with this).